### PR TITLE
Add Portugues translation for FastAPI-People

### PR DESCRIPTION
**Pull request by italopenaforte**

### CODE REVIEW

File: docs/pt/docs/fastapi-people.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# FastAPI Pessoal
+
+FastAPI has an amazing community that welcomes people from all backgrounds.
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:3]: Remove the English part, please.

File: docs/pt/docs/fastapi-people.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+# FastAPI Pessoal
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:1]: It doesn't sound as cool as in English, what do you think about: `Agentes FastAPI`? :sunglasses:

File: docs/pt/docs/fastapi-people.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Respostas: {{ user.answers }}</div><div class="count">Pull Requests: {{ user.prs }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+Eu sou o criador e mantenedor do **FastAPI**. Você pode ler mais sobre isso em [Ajuda FastAPI - Obtenha ajuda - Conecte-se com o autor](help-fastapi.md#connect-with-the-author){.internal-link target=_blank}.
+
+...Mas aqui eu quero mostrar a você a comunidade.
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:24]:
```suggestion
...Mas aqui eu quero mostrar a nossa comunidade.
```

File: docs/pt/docs/fastapi-people.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+---
+
+**FastAPI** recebe muito apoio da comunidade. E quero destacar suas contribuições.
+
+Estas são as pessoas que:
+
+* [Ajude outras pessoas com problemas (perguntas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:32]:
```suggestion
* [Ajudam outras pessoas com problemas (perguntas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.
```

File: docs/pt/docs/fastapi-people.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+* [Crie Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:33]:
```suggestion
* [Criam Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
```

File: docs/pt/docs/fastapi-people.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+* Revise Pull Requests, [especially important for translations](contributing.md#translations){.internal-link target=_blank}.
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:34]:
```suggestion
* Revisam Pull Requests, [importante especialmente para as traduções](contributing.md#translations){.internal-link target=_blank}.
```

File: docs/pt/docs/fastapi-people.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+</div>
+{% endif %}
+
+## Especialistas
+
+Aqui estão os especialistas **FastAPI**. 🤓
+
+ 
+Estes são os usuários que mais [ajudaram outras pessoas com problemas (perguntas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} through *all time*.
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:57]:
```suggestion
Estes são os usuários que mais [ajudaram outras pessoas com problemas (perguntas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} desde *sempre*.
```

File: docs/pt/docs/fastapi-people.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+They have proven to be experts by helping many others. ✨
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:59]: It is in English :P

File: docs/pt/docs/fastapi-people.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+<div class="user-list user-list-center">
+{% for user in people.experts %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Top Contributors
```
Comment (Kludex) [inline docs/pt/docs/fastapi-people.md:71]: This is not translated...

Comment (italorangel) [reply docs/pt/docs/fastapi-people.md:71]: i will commit the end of translations, sorry

Comment (Kludex) [reply docs/pt/docs/fastapi-people.md:71]: No need for apologies :smile:

File: docs/pt/docs/fastapi-people.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Pull Requests: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+Existem muitos outros contribuidores (mais de uma centena), você pode vê-los todos na 
+
+Existem muitos outros contribuidores (mais de cem), você pode vê-los todos na <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">página FastAPI GitHub Contributors</a>. 👷
```
Comment (lsglucas) [inline docs/pt/docs/fastapi-people.md:90]: I believe that this part is duplicated.

```suggestion
Existem muitos outros contribuidores (mais de cem), você pode vê-los todos na <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">página FastAPI GitHub Contributors</a>. 👷
```

File: docs/pt/docs/fastapi-people.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+
+## Sobre os dados - detalhes técnicos
+
+A intenção desta página é destacar o esforço da comunidade para ajudar os outros.
+
+Especialmente incluindo esforços que normalmente são menos visíveis e, em muitos casos, mais árduos, como ajudar outras pessoas com problemas e revisar Pull Requests com traduções.
+
+Os dados são calculados a cada mês, você pode ler o <a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">código-fonte aqui</a>.
+
+Eu também me reservo o direito de atualizar o algoritmo, seções, limites, etc (apenas no caso 🤷).
```
Comment (lsglucas) [inline docs/pt/docs/fastapi-people.md:138]: `apenas no caso` didn't make sense to me, I believe that the following suggestion makes it better.  

```suggestion
Eu também me reservo o direito de atualizar o algoritmo, seções, limites, etc (só por garantia 🤷).
```

**Review by stlucasgarcia — APPROVED** [review]

**Comment by lsglucas** [comment]
@italorangel Don't forget to add the file to `docs/pt/mkdocs.yml`

**Pull request closed without merging**